jetty-http/src/main/java/org/eclipse/jetty/http/CompressedContentFormat.java

```diff
@@ -50,7 +50,12 @@ public boolean equals(Object o)
         if (_extension == null && ccf._extension != null)
             return false;
 
-        return _encoding.equalsIgnoreCase(ccf._encoding) && _extension.equalsIgnoreCase(ccf._extension);
+        return equalsIgnoreCase(_encoding, ccf._encoding) && equalsIgnoreCase(_extension, ccf._extension);
+    }
+
+    private static boolean equalsIgnoreCase(String a, String b)
```

Comment:
Isn't there a `StringUtil.equalsIgnoreCase` method?  If not, this method should go there.